SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Eiffel-community vulnerability and security reporting and response
+
+## Report a Vulnerability or Security issue
+We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers. All reported and fixed security and vulnerability issues can be found on the [eiffel community security page](https://eiffel-community.github.io/security.html) .
```

Comment:
```suggestion
We as a community encourage researchers, users and contributors to report vulnerabilities and security related issues to the Eiffel community. All issues are thoroughly investigated by a community security officer and/or other community security volunteers. All reported and fixed security and vulnerability issues can be found on the [eiffel community security page](https://eiffel-community.github.io/security.html) .
```

Reply:
Github wouldn't let me commit the suggestions using the convinence-button, added your updates in a separate commit.